CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorConfigurationRequestHandler.swift

```diff
@@ -194,6 +269,21 @@ public class ReplicatorConfigurationRequestHandler {
             if let auto_purge: String = args.get(name: "auto_purge") {
                 config.enableAutoPurge = auto_purge.lowercased() == "enabled"
             }
+            if (collections != nil){
+                if(collection_configuration != nil){
+                    if(collection_configuration!.count == 1){
+                        config.addCollections(collections!, config: collection_configuration?[0])
+                    }
+                    else if(collection_configuration!.count > 1 && collection_configuration?.count == collections?.count ) {
+                        for (collection, configuration) in zip(collections!,collection_configuration!) {
+                            config.addCollection(collection, config: configuration)
+                        }
+                    }
+                }
+                else {
+                    config.addCollections(collections!)
+                }
+            }
```

Comment:
something like this instead of `zip`
```
if let cols = collections {
            if collection_configuration.count == 1 {
                let colConfig = collection_configuration[0]
                config.addCollections(cols, colConfig)
            } else {
                assert(collection_configuration.count == cols.count)
                for (i, colConfig) in collection_configuration.enumerated() {
                    config.addCollection(cols[i], colConfig)
                }
            }
        }
        ```

Reply:
Will using zip create any problems for the executor?